### ISSUE TITLE
Possible to open multiple Settings Views

### ISSUE THREAD

**Issue by mnquintana** [atom/settings-view#506]
There should only ever be one Settings View tab per window, and usually that's enforced, but I've run across some weird cases where you can actually open two Settings Views:

![untitled](https://cloud.githubusercontent.com/assets/823545/7660363/444810ce-fb17-11e4-9485-14535d76d018.gif)

However, I can only have 2 instances of Settings View open in this case, no more.

I'm not 100% sure about repro steps, but I think this is possible if an open Settings tab was serialized for a project, and then you reopen that project and enter the Settings View keybinding.

Atom 0.199.0
OS X 10.10.3

**Comment by mnquintana** [comment]
Hmmm, I don't think that's it. Here's what I tried:

1. Open a directory in Atom with `atom .`
2. Open a file from that directory within Atom
3. Open Settings View
4. Close the window
5. Reopen the directory with `atom .`
6. Try entering the Settings View keybinding
7. Nothing happens

Not sure how to repro this :confused:

**Comment by 50Wliu** [comment]
I just reproduced this.
1.  Open up Atom with an outdated package (so that the new notification appears)
2.  Open up Settings View (in my case, I went to the Install tab)
3.  Click on the notification
4.  New Settings tab! :frowning:

**Comment by mnquintana** [comment]
Here's the easiest way to reproduce this consistently:

1. Open Settings View
1. Split Right
1. You have two Settings Views!
1. You can also move the Settings Views to the same pane.

**Comment by olmokramer** [comment]
Why should there only be 1 instance of the settings view at any time? There have been numerous times I wanted a second settings view, but I couldn't. For example, after I've updated multiple packages, I want to read each packages changelog. When I click a package in the updater, it takes me to the packages detail page, but when I return to the updater, the list of updated packages is gone and it's searching again for new packages. If I could have a second settings view in another pane, I could use that one to check the changelog of each updated package without having to remember/write down the packages that were updated.

Well, maybe that warrants its own bug report, but I can imagine that there are scenarios where it would be useful to have multiple settings views. What are the arguments against it?
